CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.0.0 FATAL_ERROR)
+
+project(Server)
+
+#COMPILE WITH C++ 11
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+
+option(UnitTest "UnitTest" OFF)
+
+# Instruct CMake to run moc automatically when needed.
+set(CMAKE_AUTOMOC ON)
+
+find_package(Qt5Core REQUIRED)
+find_package(Qt5Network REQUIRED)
+find_package(Qt5Test REQUIRED)
+
+file(GLOB API_SRC
+  "API/*.h"
+  "API/*.cpp"
+  "API/DebianMining/*.h"
+  "API/DebianMining/*.cpp"
+  "API/Windows7Mining/*.cpp"
+  "API/Windows7Mining/*.h"
+  "API/Windows10Mining/*.cpp"
+  "API/Windows10Mining/*.h"
+  "controller/*.h"
+  "controller/*.cpp"
+  )
+
+file(GLOB SRV_SRC
```

Comment:
It makes more sense to me to have a variable for `"test"` sources, and an other for the server runtime sources (including API sources)